web/models.py

```
@@ -2805,3 +2805,135 @@ class ScheduledPost(models.Model):
+
+class MassClassPoll(models.Model):
+    """Model for polls created during a mass class stream"""
+
+    stream = models.ForeignKey(MassClassStream, on_delete=models.CASCADE, related_name="polls")
+    poll_id = models.UUIDField(unique=True)
+    question = models.TextField()
+    is_active = models.BooleanField(default=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        ordering = ["-created_at"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)